src/java.base/share/classes/java/util/jar/JarOutputStream.java

```diff
@@ -78,8 +78,12 @@ public JarOutputStream(OutputStream out) throws IOException {
      * to the start of the entry data. This method will also close
      * any previous entry. The default compression method will be
      * used if no compression method was specified for the entry.
-     * The current time will be used if the entry has no set modification
-     * time.
+     * When writing a compressed (i.e. {@code ZipEntry.DEFLATED}))
```

Comment:
I would leave as DEFLATED given it is used this way in other ZipOutputStream methods and I would also remove the  "i.e."

Reply:
I would prefer to keep it as "When writing as compressed (DEFLATED) ..." to keep the terminology consistent when linking to the setCompressedMethod.

Reply:
Agree, no need for ZipEntry.DEFLATED which is what I was trying to say (sorry if that was not clear).  Only suggesting ZipEntry.DEFLATED -> DEFLATED and remove  the ".i.e."